### PR TITLE
developers/networks: update current network plans

### PR DESCRIPTION
**Pull request by q9f**
removed kovan and ropsten, ref #8773

deprecated goerli

emphasized sepolia

### CODE REVIEW

File: src/content/developers/docs/networks/index.md [hunk 1]
```diff
@@ -28,13 +28,15 @@ In addition to Mainnet, there are public testnets. These are networks used by pr
...
 
 ETH on testnets has no real value; therefore, there are no markets for testnet ETH. Since you need ETH to actually interact with Ethereum, most people get testnet ETH from faucets. Most faucets are webapps where you can input an address which you request ETH to be sent to.
 
 #### Which Testnet should I use?
 
-The two public proof-of-stake testnets (which client developers are maintaining post-merge) are Goerli and Sepolia. The Goerli network was merged with the Prater Beacon Chain testnet. The Sepolia network was created to test the transition to proof-of-stake.
+The two public testnets which client developers are maintaining post-merge are Goerli and Sepolia. The Goerli network is targeting protocol developers to test network upgrades and stakers to test running validators. Sepolia, on the other hand, is a permissioned network targeting contract and application developers.
```
Comment (Pandapip1) [inline src/content/developers/docs/networks/index.md:37]: How about swapping the order?
```suggestion
The two public testnets that client developers are currently maintaining are Sepolia and Goerli. Sepolia is a network for contract and application developers to test their applications. The Goerli network lets protocol developers test network upgrades, and lets stakers test running validators.
```

File: src/content/developers/docs/networks/index.md [hunk 2]
```diff
@@ -43,100 +45,81 @@ The Sepolia network uses a permissioned validator set. It's fairly new, meaning
...
+**Resources**:
 
 - [Website](https://sepolia.dev/)
-- [GitHub](https://github.com/goerli/sepolia)
+- [GitHub](https://github.com/eth-clients/sepolia)
 - [Otterscan](https://sepolia.otterscan.io/)
 - [Etherscan](https://sepolia.etherscan.io)
 
-##### Sepolia faucets
+**Faucets**:
```
Comment (Pandapip1) [inline src/content/developers/docs/networks/index.md:55]:
```suggestion
##### Faucets
```
Using bold for headings is an antipattern

File: src/content/developers/docs/networks/index.md [hunk 3]
```diff
@@ -43,100 +45,81 @@ The Sepolia network uses a permissioned validator set. It's fairly new, meaning
...
-The Goerli network is open for users wanting to run a testnet validator. Stakers wanting to test protocol upgrades before they are deployed to mainnet should therefore use Goerli.
-
-- Open validator set, stakers can test network upgrades
-- Large state, useful for testing complex smart contract interactions
-- Longer to sync and requires more storage to run a node
-
-#### Sepolia {#sepolia}
-
-Sepolia is a proof-of-stake testnet, and is the recommended default testnet for application development.
+**Resources**:
```
Comment (Pandapip1) [inline src/content/developers/docs/networks/index.md:48]:
```suggestion
##### Resources
```
Using bold for headings is an antipattern

File: src/content/developers/docs/networks/index.md [hunk 4]
```diff
@@ -43,100 +45,81 @@ The Sepolia network uses a permissioned validator set. It's fairly new, meaning
...
+_Note: [the Goerli testnet is deprecated](https://ethereum-magicians.org/t/proposal-predictable-ethereum-testnet-lifecycle/11575/17) and will be replaced by [Holesovice](https://github.com/eth-clients/holesovice) in 2023. Please consider migrating your applications to Sepolia._
+
+[Goerli](#goerli) is testnet for testing of validating and staking  .
+The Goerli network is open for users wanting to run a testnet validator. Stakers wanting to test protocol upgrades before they are deployed to mainnet should therefore use Goerli.
+
+- Open validator set, stakers can test network upgrades
+- Large state, useful for testing complex smart contract interactions
+- Longer to sync and requires more storage to run a node
+
+**Resources**:
```
Comment (Pandapip1) [inline src/content/developers/docs/networks/index.md:74]:
```suggestion
##### Resources
```
Using bold for headings is an antipattern

File: src/content/developers/docs/networks/index.md [hunk 5]
```diff
@@ -43,100 +45,81 @@ The Sepolia network uses a permissioned validator set. It's fairly new, meaning
...
 
 - [Website](https://goerli.net/)
-- [GitHub](https://github.com/goerli/testnet)
+- [GitHub](https://github.com/eth-clients/goerli)
 - [Etherscan](https://goerli.etherscan.io)
 
-##### Goerli faucets
+**Faucets**:
```
Comment (Pandapip1) [inline src/content/developers/docs/networks/index.md:80]:
```suggestion
##### Faucets
```
Using bold for headings is an antipattern

File: src/content/developers/docs/networks/index.md [hunk 6]
```diff
@@ -43,100 +45,81 @@ The Sepolia network uses a permissioned validator set. It's fairly new, meaning
...
-#### Rinkeby _(deprecated)_ {#rinkeby}
+#### Rinkeby _(long-term support)_ {#rinkeby}
 
-_Note: [the Rinkeby testnet is deprecated](https://github.com/ethereum/pm/issues/460) and will no longer receive protocol upgrades. Please consider migrating your applications to Sepolia or Goerli._
+_Note: [the Rinkeby testnet is deprecated](https://blog.ethereum.org/2022/11/30/ropsten-shutdown-announcement) and will no longer receive protocol upgrades. Please consider migrating your applications to Sepolia._
 
 A proof-of-authority testnet for those running old versions of the Geth client.
 
-##### Rinkeby faucets
+**Faucets**:
```
Comment (Pandapip1) [inline src/content/developers/docs/networks/index.md:97]:
```suggestion
##### Faucets
```
Using bold for headings is an antipattern

File: src/content/developers/docs/networks/index.md [hunk 7]
```diff
@@ -43,100 +45,81 @@ The Sepolia network uses a permissioned validator set. It's fairly new, meaning
...
 
 [Layer 2 (L2)](/layer-2/) is a collective term to describe a specific set of Ethereum scaling solutions. A layer 2 is a separate blockchain that extends Ethereum and inherits the security guarantees of Ethereum. Layer 2 testnets are usually tightly coupled to public Ethereum testnets.
 
-#### Arbitrum Rinkeby {#arbitrum-rinkeby}
+#### Arbitrum Goerli {#arbitrum-goerli}
 
 A testnet for [Arbitrum](https://arbitrum.io/).
 
-Arbitrum Rinkeby faucets:
+**Faucets**:
```
Comment (Pandapip1) [inline src/content/developers/docs/networks/index.md:112]:
```suggestion
##### Faucets
```
Using bold for headings is an antipattern

File: src/content/developers/docs/networks/index.md [hunk 8]
```diff
@@ -43,100 +45,81 @@ The Sepolia network uses a permissioned validator set. It's fairly new, meaning
...
 - [Chainlink faucet](https://faucets.chain.link/)
 - [Paradigm faucet](https://faucet.paradigm.xyz/)
 
-#### Optimistic Kovan {#optimistic-kovan}
+#### Optimistic Goerli {#optimistic-goerli}
 
 A testnet for [Optimism](https://www.optimism.io/).
 
-Optimistic Kovan faucets:
+**Faucets**:
```
Comment (Pandapip1) [inline src/content/developers/docs/networks/index.md:122]:
```suggestion
##### Faucets
```
Using bold for headings is an antipattern

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

**Review by Pandapip1 — APPROVED** [review]
LGTM.

File: src/content/developers/docs/networks/index.md [hunk 9]
```diff
@@ -28,13 +28,15 @@ In addition to Mainnet, there are public testnets. These are networks used by pr
 
 You should test any contract code you write on a testnet before deploying to Mainnet. Among dapps that integrate with existing smart contracts, most projects have copies deployed to testnets.
 
-Most testnets started by using a proof-of-authority consensus mechanism. This means a small number of nodes are chosen to validate transactions and create new blocks – staking their identity in the process. Alternatively, some testnets started off using a proof-of-work consensus mechanism with just a few permissioned miners. However, in preparation for [The Merge](/upgrades/merge), these testnets underwent their own transitions to proof-of-stake, offering the opportunity for multiple 'dress-rehearsals' before developers merged Ethereum Mainnet. The Ethereum testnets are now proof-of-stake, just like Ethereum Mainnet.
+Most testnets started by using a permissioned proof-of-authority consensus mechanism. This means a small number of nodes are chosen to validate transactions and create new blocks – staking their identity in the process. Alternatively, some testnets feature an open proof-of-stake consensus mechanism where everyine can test running a validator, just like Ethereum Mainnet.
```
Comment (corwintines) [inline src/content/developers/docs/networks/index.md:31]:
```suggestion
Most testnets started by using a permissioned proof-of-authority consensus mechanism. This means a small number of nodes are chosen to validate transactions and create new blocks – staking their identity in the process. Alternatively, some testnets feature an open proof-of-stake consensus mechanism where everyone can test running a validator, just like Ethereum Mainnet.
```

**Review by corwintines — APPROVED** [review]
Thanks for these updates @q9f!

**Pull request merged**